frontend/src/reducers/queries.ts

```diff
@@ -0,0 +1,216 @@
+import { IQuery } from '../common/models/apps/querys';
+import { Routine } from 'redux-saga-routines';
+import {
+  duplicateSelectQueryRoutine,
+  fetchQueryRoutine,
+  openQueryRoutine,
+  setNewCodeRoutine,
+  setNewConfirmRoutine,
+  setNewNameQueryRoutine,
+  setNewRunRoutine,
+  setSelectQueryRoutine, setSuccessTriggersRoutine,
+  setUnSuccessTriggersRoutine,
+  setWaiterQueryRoutine
+} from '../scenes/constructor/routines';
+import { ITrigger } from '../common/models/query/ITrigger';
+
+export interface IQueryState {
+  queriesApp: Array<IQuery>;
+  selectQuery:{
+    selectQueryId: string,
+    selectQueryName: string,
+    selectQueryCode: string,
+    selectQueryTriggers:Array<ITrigger>|[],
+    runAutomatically: boolean,
+    showConfirm: boolean
+  },
+  setNewCode:string,
+  setNewRun: boolean,
+  runAutomaticallyTitle:string,
+  setNewName:string,
+  setNewConfirm:boolean,
+  setNewSuccessTriggers:Array<ITrigger>|[],
+  setNewUnSuccessTriggers:Array<ITrigger>|[],
+  queriesAppLength:number,
+  isLoading: boolean,
+  isOpen:boolean,
+  isDuplicate:boolean,
+  waitingQuery:{
+    QueryId?:string,
+    QueryName?:string,
+    QueryCode?:string,
+    QueryTriggers:Array<ITrigger>|[],
```

Comment:
why do we need to capitalize the first letter here? 
Also move `waitingQuery` type to models